major_event_log/feeds.py

```diff
@@ -21,8 +21,10 @@ class LatestEventsFeed(Feed):
     feed_type = MajorEventLogFeed
     # Required tags by the Atom feed.
     title = "PREMIS Major Event Log"
-    link = "/major-event-log/"
+    link = reverse_lazy("major-event-log:index")
```

Comment:
If I do that then the first few lines of the feed look like this:
```
<feed xmlns="http://www.w3.org/2005/Atom" xml:lang="en-us">
<title>PREMIS Major Event Log</title>
<link href="http://127.0.0.1:8000/major-event-log/feed/" rel="alternate"/>
<link href="http://127.0.0.1:8000/major-event-log/feed/" rel="self"/>
<id>http://127.0.0.1:8000/major-event-log/feed/</id>
```
Instead of this:
```
<feed xmlns="http://www.w3.org/2005/Atom" xml:lang="en-us">
<title>PREMIS Major Event Log</title>
<link href="http://127.0.0.1:8000/major-event-log/" rel="alternate"/>
<link href="http://127.0.0.1:8000/major-event-log/feed/" rel="self"/>
<id>http://127.0.0.1:8000/major-event-log/</id>
```
Seems weird to have two links to the exact same place. But I'll make that change if you want it.